### PR TITLE
User operation kickstart tests.

### PR DESCRIPTION
**Pull request by KosiehBarter**
Kickstart scripts for adding specified user during installation as well
as check-up if the user was successfully added to the system, including
password, /home/<user> directory and specified group.

Signed-off-by: Karel Valek <kvalek@redhat.com>

### CODE REVIEW

File: tests/kickstart_tests/user.ks [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+## TEST CREATE USER CHECK
+
+# First, check if the group is properly set up.
+cat /etc/group | grep 5001
+if [[ $? -ne 0 ]]; then
+    echo "FAILED: Group failed to create." >> /root/RESULT
+fi
+
+# If it succeeds, lets go to higher level. Check, if the user is present..
```
Comment (jkonecny12) [inline tests/kickstart_tests/user.ks]: Remove 'If it succeeds, lets go yo higher level.' and similar below. When you write comments use rule: shorter is better.
This information isn't mandatory for the reader.

File: tests/kickstart_tests/user.ks [hunk 2]
```diff
@@ -0,0 +1,88 @@
+# Set basic URL
+url --url="http://dl.fedoraproject.org/pub/fedora/linux/development/$releasever/$basearch/os/"
+
+# perform install
```
Comment (jkonecny12) [inline tests/kickstart_tests/user.ks]: Please use everywhere capital letters or everything with small letters. Don't mix between them.

**Comment by jkonecny12** [comment]
Other than that I think it's ok

**Comment by KosiehBarter** [comment]
FIXED: Capitalization, comments.

**Comment by KosiehBarter** [comment]
Fixed: Removed "FAILED:", removed unneeded comments.

**Comment by jkonecny12** [comment]
Looks good to me

**Comment by clumens** [comment]
Looks good to me too.  Feel free to push.

File: tests/kickstart_tests/user.ks [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+shutdown
+
+%packages
+%end
+
+%post
+
+## TEST CREATE USER CHECK
+# Check group
+cat /etc/group | grep 5001
```
Comment (bcl) [inline tests/kickstart_tests/user.ks]: May as well check for the group name too

File: tests/kickstart_tests/user.ks [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+if [[ $? -ne 0 ]]; then
+    echo "User is present, but not all details: REAL NAME (GEDOS)" >> /root/RESULT
+fi
+
+# Check if the user has his/her bash
+cat /etc/passwd | grep kosieh | grep "/bin/bash"
+if [[ $? -ne 0 ]]; then
+    echo "User is present, but /bin/bash is not set" >> /root/RESULT
+fi
+
```
Comment (bcl) [inline tests/kickstart_tests/user.ks]: Add a check to make sure the home directory is in /etc/passwd too.

File: tests/kickstart_tests/user.ks [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+install
+
+network --bootproto=dhcp
+
+bootloader --timeout=1
+zerombr
+
+clearpart --all
+autopart --type=lvm
```
Comment (bcl) [inline tests/kickstart_tests/user.ks]: This should just be autopart since we're not testing autopart type (and to be consistent with the other kickstarts)

**Comment by KosiehBarter** [comment]
Edited locally, added test for /etc/passwd home dir + group name.
Pushed.

**Pull request closed without merging**